tools/correct_line.sh

```diff
@@ -0,0 +1,27 @@
+ #!/bin/bash
```

Comment:
The name `correct_line.sh` makes it sound like this file will correct our lines. It in fact _checks_ our line _length_, so I propose naming it `check_line_length.sh`, matching your `check_style_proto.sh`.